### PR TITLE
[RM] Alternative mass balance

### PR DESCRIPTION
**Pull request by skai95**
Fixes # 

1. [x] Feature description was added to the [changelog](https://github.com/ufz/ogs/wiki/Release-notes-6.3.1)
2. [x] Tests covering your feature were added?
3. [x] Any new feature or behavior change was documented?

### CODE REVIEW

File: Tests/Data/RichardsMechanics/alternative_mass_balance_anzInterval_10.prj [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+                    <type>Dirichlet</type>
+                    <parameter>pressure_bc</parameter>
+                </boundary_condition>
+                <boundary_condition>
+                    <geometrical_set>square_1x1_geometry</geometrical_set>
+                    <geometry>right</geometry>
+                    <type>Dirichlet</type>
+                    <parameter>pressure_bc</parameter>
+                </boundary_condition>
+                <!--boundary_condition> werden automatisch neumann 0 RB
```
Comment (endJunction) [inline Tests/Data/RichardsMechanics/alternative_mass_balance_anzInterval_10.prj:269]: Drop the commented block. Comments in English please ;)

File: Tests/Data/RichardsMechanics/alternative_mass_balance_anzInterval_10.prj [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+            <type>Constant</type>
+            <value>1e9</value>
+        </parameter>
+        <parameter>
+            <name>nu</name>
+            <type>Constant</type>
+            <value>0.3</value>
+        </parameter>
+        <!-- Model parameters -->
+        <!--parameter>
```
Comment (TomFischer) [inline Tests/Data/RichardsMechanics/alternative_mass_balance_anzInterval_10.prj:175]: Can this commented parameter be removed? If not, please drop a short message here why it is necessary to keep it.

File: Tests/Data/RichardsMechanics/alternative_mass_balance_anzInterval_10.prj [hunk 3]
```diff
@@ -0,0 +1,292 @@
...
+                            <name>biot_coefficient</name>
+                            <type>Constant</type>
+                            <value>1</value>
+                        </property>
+                        <property>
+                            <name>porosity</name>
+                            <type>Constant</type>
+                            <value>0.5</value>
+                        </property>
+                        <!--property>
```
Comment (TomFischer) [inline Tests/Data/RichardsMechanics/alternative_mass_balance_anzInterval_10.prj:75]: Can this commented property be removed? If not, please drop a short message here why it is necessary to keep it.

File: Tests/Data/RichardsMechanics/RichardsFlow_2d_small_masslumping_pcs_0.pvd [hunk 4]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
```
Comment (TomFischer) [inline Tests/Data/RichardsMechanics/RichardsFlow_2d_small_masslumping_pcs_0.pvd:1]: Is this pvd-file used somewhere? Can it be deleted?

File: Tests/Data/RichardsMechanics/alternative_mass_balance_anzInterval_10_pcs_0.pvd [hunk 5]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
```
Comment (TomFischer) [inline Tests/Data/RichardsMechanics/alternative_mass_balance_anzInterval_10_pcs_0.pvd:1]: Is this pvd-file used somewhere? Can it be deleted?

**Pull request merged**